app/components/windows/AdvancedStatistics.tsx

```diff
@@ -0,0 +1,211 @@
+import { Component } from 'vue-property-decorator';
+import { Inject } from 'services/core/injector';
+import TsxComponent from 'components/tsx-component';
+import { $t } from 'services/i18n';
+import { WindowsService } from 'services/windows';
+import cx from 'classnames';
+import styles from './AdvancedStatistics.m.less';
+import ModalLayout from 'components/ModalLayout.vue';
+import PerformanceMetrics from './../PerformanceMetrics.vue';
+import { StreamingService, EStreamingState } from 'services/streaming';
+import GlobalSyncStatus from 'components/GlobalSyncStatus.vue';
+import moment from 'moment';
+import { Subscription } from 'rxjs';
+import { PerformanceService } from 'services/performance';
+import {
+  ENotificationType,
+  ENotificationSubType,
+  NotificationsService,
+  INotification,
+} from 'services/notifications';
+
+@Component({})
+export default class AdvancedStatistics extends TsxComponent<{}> {
+  @Inject() windowsService: WindowsService;
+  @Inject() notificationsService: NotificationsService;
+  @Inject() performanceService: PerformanceService;
+  @Inject() streamingService: StreamingService;
+
+  private updateInterval = 0;
+  private notifications: INotification[] = [];
+  private streamingStatus = '';
+  private notificationPushed: Subscription = null;
+  private streamingStatusChange: Subscription = null;
+
+  mounted() {
+    this.notificationPushed = this.notificationsService.notificationPushed.subscribe(notify => {
+      this.onNotificationHandler(notify);
+    });
+
+    this.streamingStatusChange = this.streamingService.streamingStatusChange.subscribe(status => {
+      this.onStreamingStatusChange(status);
+    });
+
+    this.notifications = this.notificationsService
+      .getAll()
+      .filter(notification => notification.subType !== ENotificationSubType.DEFAULT);
+    // update the time labels
+    this.updateInterval = window.setInterval(() => {
+      this.$forceUpdate();
+    }, 60 * 1000);
+  }
+
+  destroyed() {
+    this.notificationsService.markAllAsRead();
+    this.notificationPushed.unsubscribe();
+    this.streamingStatusChange.unsubscribe();
+    clearInterval(this.updateInterval);
+  }
+
+  get notificationGroups(): { unread: INotification[]; read: INotification[] } {
+    return {
+      unread: this.notificationsService.getUnread(),
+      read: this.notificationsService.getRead(),
+    };
+  }
+
+  get notificationsCount() {
+    return this.notificationsService.getAll().length;
+  }
+
+  get percentDropped() {
+    return this.performanceService.state.percentageDroppedFrames || 0;
+  }
+
+  get status(): { type: string; description: string } {
+    if (this.streamingStatus === 'reconnecting' || this.percentDropped > 50) {
+      return {
+        type: 'error',
+        description: $t('Streamlabs OBS is experiencing difficulties broadcasting'),
+      };
+    }
+
+    if (this.percentDropped > 30) {
```

Comment:
AFAIK we have  `droppedThreshold=0.1` value in the TroubleshooterService. Are we going to use another value here?

Reply:
Yeah the idea is that we don't want everyone to think their stream is going bad, and 10% dropped frames isn't necessarily going to significantly reduce the quality of the stream. Discussed with Andy and we haven't decided on final numbers yet.